### ISSUE TITLE
[BUG/ISSUE] Crash

### ISSUE THREAD

**Issue by OminousDiscovery** [CERBON-MODS/Bosses-of-Mass-Destruction-FORGE#2]
## Before submitting the issue

- [ ] **I have checked that the bug is not a duplicate or, if it is, it needs to be reopened**
- [ ] **I have checked that I am running the latest version of Bosses of Mass Destruction [FORGE] Mod**
- [ ] **I have checked the [FAQ](https://github.com/CERBON-MODS/Bosses-of-Mass-Destruction-FORGE/blob/master/FAQ.md) section for solutions**
- [ ] **I have read the [contributing guidelines](https://github.com/CERBON-MODS/Bosses-of-Mass-Destruction-FORGE/blob/master/CONTRIBUTING.md#issues) and I agree with the [Code of Conduct](https://github.com/CERBON-MODS/Bosses-of-Mass-Destruction-FORGE/blob/master/CODE_OF_CONDUCT.md)**

## Minecraft Version
1.20.1

## To Reproduce
not sure

## Expected behavior
the game launches

## Latest Log

<details>

```
---- Minecraft Crash Report ----
// Don't do that.
Time: 2023-11-28 02:12:46
Description: Initializing game
java.lang.ExceptionInInitializerError: null
	at net.minecraft.client.gui.components.BossHealthOverlay.<clinit>(BossHealthOverlay.java:18) ~[client-1.20.1-20230612.114412-srg.jar%23726!/:?] {re:mixin,pl:connector_pre_launch:A,pl:runtimedistcleaner:A,re:classloading,pl:mixin:APP:obscure_api.mixins.json:client.BossHealthOverlayMixin from mod (unknown),pl:mixin:APP:blue_skies.mixins.json:BossHealthOverlayMixin from mod (unknown),pl:mixin:APP:aether.mixins.json:client.BossHealthOverlayMixin from mod (unknown),pl:mixin:APP:mixins.bosses_of_mass_destruction.json:BossHealthOverlayMixin from mod (unknown),pl:mixin:A,pl:connector_pre_launch:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.gui.Gui.<init>:L134
	at net.minecraftforge.client.gui.overlay.ForgeGui.<init>:L69
	at net.minecraft.client.Minecraft.<init>:L521
	at net.minecraft.client.main.Main.main:L182
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0:native
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke:L77
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke:L43
	at java.lang.reflect.Method.invoke:L568
	at net.minecraftforge.fml.loading.targets.CommonLaunchHandler.runTarget:L111
	at net.minecraftforge.fml.loading.targets.CommonLaunchHandler.clientService:L99
	at net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$makeService$0:L25
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch:L30
	at cpw.mods.modlauncher.LaunchServiceHandler.launch:L53
	at cpw.mods.modlauncher.LaunchServiceHandler.launch:L71
	at cpw.mods.modlauncher.Launcher.run:L108
	at cpw.mods.modlauncher.Launcher.main:L78
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept:L26
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept:L23
	at cpw.mods.bootstraplauncher.BootstrapLauncher.main:L141
Caused by: java.lang.RuntimeException: Config 'class com.cerbon.bosses_of_mass_destruction.config.BMDConfig' has not been registered
	at me.shedaniel.autoconfig.AutoConfig.getConfigHolder:L77
	at com.cerbon.bosses_of_mass_destruction.entity.BMDEntities.<clinit>(BMDEntities.java:49) ~[BOMD-Forge-1.20.1-1.0.1.jar%23495!/:1.0.1] {re:mixin,re:classloading}
	... 20 more
A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------
-- Head --
Thread: Render thread
Stacktrace:
	at net.minecraft.client.gui.components.BossHealthOverlay.<clinit>(BossHealthOverlay.java:18) ~[client-1.20.1-20230612.114412-srg.jar%23726!/:?] {re:mixin,pl:connector_pre_launch:A,pl:runtimedistcleaner:A,re:classloading,pl:mixin:APP:obscure_api.mixins.json:client.BossHealthOverlayMixin from mod (unknown),pl:mixin:APP:blue_skies.mixins.json:BossHealthOverlayMixin from mod (unknown),pl:mixin:APP:aether.mixins.json:client.BossHealthOverlayMixin from mod (unknown),pl:mixin:APP:mixins.bosses_of_mass_destruction.json:BossHealthOverlayMixin from mod (unknown),pl:mixin:A,pl:connector_pre_launch:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.gui.Gui.<init>:L134
	at net.minecraftforge.client.gui.overlay.ForgeGui.<init>:L69
	at net.minecraft.client.Minecraft.<init>:L521
-- Initialization --
Details:
	Modules: 
		ADVAPI32.dll:Advanced Windows 32 Base API:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		COMCTL32.dll:User Experience Controls Library:6.10 (WinBuild.160101.0800):Microsoft Corporation
		CRYPT32.dll:Crypto API32:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		CRYPTSP.dll:Cryptographic Service Provider API:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		ColorAdapterClient.dll:Microsoft Color Adapter Client:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		CoreMessaging.dll:Microsoft CoreMessaging Dll:10.0.19041.3636:Microsoft Corporation
		CoreUIComponents.dll:Microsoft Core UI Components Dll:10.0.19041.3636:Microsoft Corporation
		DBGHELP.DLL:Windows Image Helper:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		DEVOBJ.dll:Device Information Set DLL:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		DNSAPI.dll:DNS Client API DLL:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		GDI32.dll:GDI Client DLL:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		GEPPlugin64.dll:GEPPlugin:1.1.0.39:Overwolf LTD
		GLU32.dll:OpenGL Utility Library DLL:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		IMM32.DLL:Multi-User Windows IMM32 API Client DLL:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		IPHLPAPI.DLL:IP Helper API:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		KERNEL32.DLL:Windows NT BASE API Client DLL:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		KERNELBASE.dll:Windows NT BASE API Client DLL:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		MMDevApi.dll:MMDevice API:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		MSCTF.dll:MSCTF Server DLL:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		MpOav.dll:IOfficeAntiVirus Module:4.18.23100.2009 (8fcbd1c22d82af16ba34560e1a70591413e88d17):Microsoft Corporation
		NLAapi.dll:Network Location Awareness 2:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		NSI.dll:NSI User-mode interface DLL:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		NTASN1.dll:Microsoft ASN.1 API:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		OLEAUT32.dll:OLEAUT32.DLL:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		OWClient.dll:Overlay:2.2.234.8:Overwolf LTD
		OWExplorer.dll:Overlay:2.2.234.3:Overwolf LTD
		OWUtils.dll:Overlay:2.2.234.3:Overwolf LTD
		OpenAL.dll:Main implementation library:1.21.1:
		POWRPROF.dll:Power Profile Helper DLL:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		PROPSYS.dll:Microsoft Property System:7.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		PSAPI.DLL:Process Status Helper:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		RPCRT4.dll:Remote Procedure Call Runtime:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		SETUPAPI.dll:Windows Setup API:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		SHCORE.dll:SHCORE:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		SHELL32.dll:Windows Shell Common Dll:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		UMPDC.dll
		USER32.dll:Multi-User Windows USER API Client DLL:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		USERENV.dll:Userenv:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		VCRUNTIME140.dll:Microsoft® C Runtime Library:14.29.30139.0 built by: vcwrkspc:Microsoft Corporation
		VERSION.dll:Version Checking and File Installation Libraries:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		WINHTTP.dll:Windows HTTP Services:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		WINMM.dll:MCI API DLL:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		WINNSI.DLL:Network Store Information RPC interface:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		WINSTA.dll:Winstation Library:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		WS2_32.dll:Windows Socket 2.0 32-Bit DLL:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		WSOCK32.dll:Windows Socket 32-Bit DLL:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		WTSAPI32.dll:Windows Remote Desktop Session Host Server SDK APIs:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		Wldp.dll:Windows Lockdown Policy:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		amsi.dll:Anti-Malware Scan Interface:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		bcrypt.dll:Windows Cryptographic Primitives Library:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		bcryptPrimitives.dll:Windows Cryptographic Primitives Library:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		cfgmgr32.dll:Configuration Manager DLL:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		clbcatq.dll:COM+ Configuration Catalog:2001.12.10941.16384 (WinBuild.160101.0800):Microsoft Corporation
		combase.dll:Microsoft COM for Windows:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		cryptbase.dll:Base cryptographic API DLL:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		cryptnet.dll:Crypto Network Related API:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		dbgcore.DLL:Windows Core Debugging Helpers:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		dhcpcsvc.DLL:DHCP Client Service:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		dhcpcsvc6.DLL:DHCPv6 Client:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		dinput8.dll:Microsoft DirectInput:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		drvstore.dll:Driver Store API:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		dwmapi.dll:Microsoft Desktop Window Manager API:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		dxcore.dll:DXCore:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		fwpuclnt.dll:FWP/IPsec User-Mode API:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		gdi32full.dll:GDI Client DLL:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		gep_minecraft.dll:TODO: <File description>:237.0.1.0:TODO: <Company name>
		glfw.dll:GLFW 3.4.0 DLL:3.4.0:GLFW
		gpapi.dll:Group Policy Client API:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		icm32.dll:Microsoft Color Management Module (CMM):10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		imagehlp.dll:Windows NT Image Helper:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		inputhost.dll:InputHost:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		java.dll:OpenJDK Platform binary:17.0.8.0:Microsoft
		javaw.exe:OpenJDK Platform binary:17.0.8.0:Microsoft
		jemalloc.dll
		jimage.dll:OpenJDK Platform binary:17.0.8.0:Microsoft
		jli.dll:OpenJDK Platform binary:17.0.8.0:Microsoft
		jna6342506062094564484.dll:JNA native library:6.1.4:Java(TM) Native Access (JNA)
		jsvml.dll:OpenJDK Platform binary:17.0.8.0:Microsoft
		jvm.dll:OpenJDK 64-Bit server VM:17.0.8.0:Microsoft
		kernel.appcore.dll:AppModel API Host:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		libowgameevents64.dll:Overwolf Game Events SDK:1.0.0.12:Overwolf LTD
		lwjgl.dll
		lwjgl_opengl.dll
		lwjgl_stb.dll
		management.dll:OpenJDK Platform binary:17.0.8.0:Microsoft
		management_ext.dll:OpenJDK Platform binary:17.0.8.0:Microsoft
		msasn1.dll:ASN.1 Runtime APIs:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		mscms.dll:Microsoft Color Matching System DLL:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		msvcp140.dll:Microsoft® C Runtime Library:14.29.30139.0 built by: vcwrkspc:Microsoft Corporation
		msvcp_win.dll:Microsoft® C Runtime Library:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		msvcrt.dll:Windows NT CRT DLL:7.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		mswsock.dll:Microsoft Windows Sockets 2.0 Service Provider:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		napinsp.dll:E-mail Naming Shim Provider:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		ncrypt.dll:Windows NCrypt Router:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		net.dll:OpenJDK Platform binary:17.0.8.0:Microsoft
		nio.dll:OpenJDK Platform binary:17.0.8.0:Microsoft
		ntdll.dll:NT Layer DLL:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		ntmarta.dll:Windows NT MARTA provider:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		nvgpucomp64.dll:NVIDIA GPU Compiler Driver, Version 546.17 :31.0.15.4617:NVIDIA Corporation
		nvoglv64.dll:NVIDIA Compatible OpenGL ICD:31.0.15.4617:NVIDIA Corporation
		nvspcap64.dll:NVIDIA Game Proxy:3.27.0.120:NVIDIA Corporation
		ole32.dll:Microsoft OLE for Windows:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		opengl32.dll:OpenGL Client DLL:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		pdh.dll:Windows Performance Data Helper DLL:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		perfos.dll:Windows System Performance Objects DLL:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		perfproc.dll:Windows System Process Performance Objects DLL:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		pnrpnsp.dll:PNRP Name Space Provider:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		profapi.dll:User Profile Basic API:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		rasadhlp.dll:Remote Access AutoDial Helper:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		rsaenh.dll:Microsoft Enhanced Cryptographic Provider:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		sechost.dll:Host for SCM/SDDL/LSA Lookup APIs:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		shlwapi.dll:Shell Light-weight Utility Library:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		sunmscapi.dll:OpenJDK Platform binary:17.0.8.0:Microsoft
		textinputframework.dll:"TextInputFramework.DYNLINK":10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		ucrtbase.dll:Microsoft® C Runtime Library:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		uxtheme.dll:Microsoft UxTheme Library:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		vcruntime140_1.dll:Microsoft® C Runtime Library:14.29.30139.0 built by: vcwrkspc:Microsoft Corporation
		verify.dll:OpenJDK Platform binary:17.0.8.0:Microsoft
		win32u.dll:Win32u:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		windows.storage.dll:Microsoft WinRT Storage API:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		winrnr.dll:LDAP RnR Provider DLL:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		wintrust.dll:Microsoft Trust Verification APIs:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		wintypes.dll:Windows Base Types DLL:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		wshbth.dll:Windows Sockets Helper DLL:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		xinput1_4.dll:Microsoft Common Controller API:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		zip.dll:OpenJDK Platform binary:17.0.8.0:Microsoft
Stacktrace:
	at net.minecraft.client.main.Main.main:L182
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0:native
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke:L77
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke:L43
	at java.lang.reflect.Method.invoke:L568
	at net.minecraftforge.fml.loading.targets.CommonLaunchHandler.runTarget:L111
	at net.minecraftforge.fml.loading.targets.CommonLaunchHandler.clientService:L99
	at net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$makeService$0:L25
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch:L30
	at cpw.mods.modlauncher.LaunchServiceHandler.launch:L53
	at cpw.mods.modlauncher.LaunchServiceHandler.launch:L71
	at cpw.mods.modlauncher.Launcher.run:L108
	at cpw.mods.modlauncher.Launcher.main:L78
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept:L26
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept:L23
	at cpw.mods.bootstraplauncher.BootstrapLauncher.main:L141
-- System Details --
Details:
	Minecraft Version: 1.20.1
	Minecraft Version ID: 1.20.1
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 17.0.8, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 870017536 bytes (829 MiB) / 26[004](https://gnomebot.dev/paste/1178956742705553418#L4)68480 bytes (2480 MiB) up to 12582912000 bytes (12000 MiB)
	CPUs: 12
	Processor Vendor: GenuineIntel
	Processor Name: Intel(R) Core(TM) i5-10400 CPU @ 2.90GHz
	Identifier: Intel64 Family 6 Model 165 Stepping 3
	Microarchitecture: unknown
	Frequency (GHz): 2.90
	Number of physical packages: 1
	Number of physical CPUs: 6
	Number of logical CPUs: 12
	Graphics card #0 name: NVIDIA GeForce GTX 1660 SUPER
	Graphics card #0 vendor: NVIDIA (0x10de)
	Graphics card #0 VRAM (MB): 4095.00
	Graphics card #0 deviceId: 0x21c4
	Graphics card #0 versionInfo: DriverVersion=31.0.15.4617
	Memory slot #0 capacity (MB): 32768.00
	Memory slot #0 clockSpeed (GHz): 2.67
	Memory slot #0 type: DDR4
	Virtual memory max (MB): 37492.50
	Virtual memory used (MB): 19204.12
	Swap memory total (MB): 4864.00
	Swap memory used (MB): 156.78
	JVM Flags: 4 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx12000m -Xms256m
	Loaded Shaderpack: (off)
	Launched Version: forge-47.1.3
	Backend library: LWJGL version 3.3.1 build 7
	Backend API: NVIDIA GeForce GTX 1660 SUPER/PCIe/SSE2 GL version 4.6.0 NVIDIA 546.17, NVIDIA Corporation
	Window size: <not initialized>
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'forge'
	Type: Client (map_client.txt)
	CPU: 12x Intel(R) Core(TM) i5-10400 CPU @ 2.90GHz
	Sinytra Connector: 1.0.0-beta.27+1.20.1
		SINYTRA CONNECTOR IS PRESENT!
		Please verify issues are not caused by Connector before reporting them to mod authors. If you're unsure, file a report on Connector's issue tracker.
		Connector's issue tracker can be found at https://github.com/Sinytra/Connector/issues.
	ModLauncher: 10.0.9+10.0.9+main.dcd20f30
	ModLauncher launch target: forgeclient
	ModLauncher naming: srg
	ModLauncher services: 
		mixin-0.8.5.jar mixin PLUGINSERVICE 
		eventbus-6.0.5.jar eventbus PLUGINSERVICE 
		fmlloader-1.20.1-47.1.3.jar slf4jfixer PLUGINSERVICE 
		fmlloader-1.20.1-47.1.3.jar object_holder_definalize PLUGINSERVICE 
		fmlloader-1.20.1-47.1.3.jar runtime_enum_extender PLUGINSERVICE 
		fmlloader-1.20.1-47.1.3.jar capability_token_subclass PLUGINSERVICE 
		accesstransformers-8.0.4.jar accesstransformer PLUGINSERVICE 
		fmlloader-1.20.1-47.1.3.jar runtimedistcleaner PLUGINSERVICE 
		modlauncher-10.0.9.jar mixin TRANSFORMATIONSERVICE 
		modlauncher-10.0.9.jar fml TRANSFORMATIONSERVICE 
		modlauncher-10.0.9.jar mixin-transmogrifier TRANSFORMATIONSERVICE 
		modlauncher-10.0.9.jar connector_loader TRANSFORMATIONSERVICE 
	FML Language Providers: 
		javafml@null
		kotlinforforge@4.7.0
		lowcodefml@null
		minecraft@1.0
		gml@4.0.9
	Mod List: 
		client-1.20.1-20230612.114412-srg.jar             |Minecraft                     |minecraft                     |1.20.1              |NONE      |Manifest: a1:d4:5e:04:4f:d3:d6:e0:7b:37:97:cf:77:b0:de:ad:4a:47:ce:8c:96:49:5f:0a:cf:8c:ae:b2:6d:4b:8a:3f
		forge-1.20.1-47.1.3-universal.jar                 |Forge                         |forge                         |47.1.3              |NONE      |Manifest: 84:ce:76:e8:45:35:e4:0e:63:86:df:47:59:80:0f:67:6c:c1:5f:6e:5f:4d:b3:54:47:1a:9f:7f:ed:5e:f2:90
```

</details>

**Comment by CerbonXD** [comment]
I see that you are using Sinytra Connector, please uninstall it and test if the mod works. In case it does not work please let me know. Also, If you don't have installed Cloth Config API and Geckolib mods, please install them.

**Comment by CerbonXD** [comment]
Did you test everything? Is the mod working?

**Comment by CerbonXD** [comment]
Fixed on version 1.0.3
